### ISSUE TITLE
uHAL yum install doesn't create /opt/cactus folder 

### ISSUE THREAD

**Issue by yotarid** [ipbus/ipbus-software#266]
Dear developers, 

I am working on a CC7 machine, and trying to install uHAL via YUM following the instructions you provide on your webpage. When running the last requested command "yum groupinstall uhal controlhub" , yum states that the installation was successful. However, at the bottom of the webpage, you state that the RPMs install all of their files in /opt/cactus folder, but I don't get this folder after my yum install. 
Do you have any idea what's the reason behind it ? 

Many thanks in advance.
Cheers,
Younes

**Comment by tswilliams** [comment]
Hi,

Can you send me the output of the following command:
```
rpm -qa | grep -i cactuscore | sort
```

Cheers,
Tom

**Comment by yotarid** [comment]
Hi, 

I fixed the issue. I cleaned up all uhal installed packages the following way
yum list installed | grep uhal 
yum remove packages_from_previous_commands
I then restarted a fresh install using yum and following the instructions on the website. This time uhal is properly installed and the /opt/cactus folder is correctly created. 

Sorry for my noise. I suspect that the residual uhal packages from a previous installation interfere with the newer installation and prevent it from creating the /opt/cactus folder.

Cheers,
Younes
